sswlib/src/main/java/Print/PrintVehicle.java

```diff
@@ -414,7 +414,8 @@ private void DrawMechData( Graphics2D graphics ) {
 
         //Cost
         graphics.setFont( PrintConsts.SmallFont );
-        graphics.drawString( String.format( "%1$,.0f C-Bills", Math.floor( CurVee.GetTotalCost() + 0.5f ) ), p[PrintConsts.COST].x, p[PrintConsts.COST].y );
+        graphics.drawString( String.format("%1$,.0f C-Bills", CurVee.GetTotalCost()), p[PrintConsts.COST].x, p[PrintConsts.COST].y);
+        graphics.drawString( String.format("+%1$,.0f (ammo)", CurVee.GetAmmoCosts()), p[PrintConsts.AMMO].x, p[PrintConsts.AMMO].y);
```

Comment:
This does not handle VTOLs, so the VTOL output will not match the Ground Vehicles and 'Mechs.

Reply:
Ah, didn't catch that. I will go and update VTOL's.

Reply:
Now that i think about it, this probably doesn't cover naval as well.

Reply:
Apparently VTOL and Naval count as CombatVechiles so they would have output but not correctly and if someone had tried to print the PDF it would have given them an out of bounds exception.